docs/api/signer.rst

```diff
@@ -1,16 +1,34 @@
 Signer
 ======
 
+--------------------
+BaseSigner interface
+--------------------
+
 .. py:module:: starknet_py.net.signer
 
 .. autoclass:: BaseSigner
     :members:
+    :member-order: groupwise
+
+---------------------------------
+BaseSigner default implementation
+---------------------------------
 
 By default, StarkNet.py uses ``StarkCurveSigner`` which works with OpenZeppelin's account contract.
 
 .. py:module:: starknet_py.net.signer.stark_curve_signer
 
 .. autoclass:: StarkCurveSigner
+    :members:
+    :member-order: groupwise
+
+-------
+KeyPair
+-------
 
 .. autoclass:: KeyPair
+    :members:
+    :undoc-members:
```

Comment:
Was this omitted on purpose?

Reply:
Yes, there is no need to add docstring for `private_key` and `public_key`, but would be nice to have them in the docs